### PR TITLE
TIL about HTML5 support in the `tidy` utility

### PR DESCRIPTION
**Pull request by mjankowski**

### CODE REVIEW

File: html/html5-tidy.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Tidy with HTML5 support
+
+There is a tool called `tidy` which was originally written by [Dave
+Raggett][tidy] (in the late '90s), and is now [maintained by
+volunteers][tidy-sf] and the W3C.
+
+There is an experimental fork of `tidy` which is maintained by the HTML Tidy
+Advocacy Community Group which supports the modern HTML5 spec.
+
+The tool is used by giving it some HTML/XML/XHTML as input, and producing a
```
Comment (JoelQ) [inline html/html5-tidy.md]: How about:

> This tool takes HTML/XML/XHTML as input and produces a valid version as it's output.

**Comment by JoelQ** [comment]
LGTM :rocket:  Merge and buffer at your leisure.

File: html/html5-tidy.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Tidy with HTML5 support
+
+There is a tool called `tidy` which was originally written by [Dave
+Raggett][tidy] (in the late '90s), and is now [maintained by
+volunteers][tidy-sf] and the W3C.
+
+There is an experimental fork of `tidy` which is maintained by the HTML Tidy
+Advocacy Community Group which supports the modern HTML5 spec.
+
+This tool takes HTML/XML/XHTML as input and produces a valid version as it's
```
Comment (mike-burns) [inline html/html5-tidy.md]: its

File: html/html5-tidy.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+output. It also has an errors mode where it will generate warnings and errors,
```
Comment (mike-burns) [inline html/html5-tidy.md]: Would it more descriptive to call this a "lint mode"?

**Pull request merged**